project/src/main/world/career-world.gd

```diff
@@ -123,7 +123,10 @@ func _fill_environment_scene() -> void:
 		var player := overworld_environment.add_creature()
 		player.creature_id = CreatureLibrary.PLAYER_ID
 	
-	if not _find_sensei():
+	if PlayerData.career.current_region().sensei_absent:
+		# don't add the sensei
+		pass
+	elif not _find_sensei():
 		var sensei := overworld_environment.add_creature()
 		sensei.creature_id = CreatureLibrary.SENSEI_ID
```

Comment:
This block doesn't make sense, it should just be an 'if else'

Reply:
Changed to `if... else, if` for clarity and symmetry. (The functionality did not change.)